subworkflows/local/alignment.nf

```diff
@@ -13,17 +13,17 @@ include { SAMTOOLS_VIEW        } from '../../modules/nf-core/samtools/view/main'
 
 workflow ALIGNMENT {
     take:
-        reads
-        star_index
-        gtf
-        platform
-        subsample_bed
-        seed_frac
-        num_reads
-        switch_subsample_region
-        switch_downsample
-        salmon_index
-        ch_genome_fasta
+        reads                   // channel: [mandatory] [ val(meta), [path(reads)]  ]
+        star_index              // channel: [mandatory] [ val(meta), path(star_index) ]
+        ch_gtf                  // channel: [mandatory] [ val(meta), path(gtf) ]
+        platform                // channel: [ val(platform) ]
+        subsample_bed           // patrameter: [ path(subsample_bed) ]
+        seed_frac               // patrameter: [ val(seed_frac) ]
+        num_reads               // patrameter: [ val(num_reads) ]
+        switch_subsample_region // patrameter: [ val(switch_subsample_region) ]
+        switch_downsample       // patrameter: [ val(switch_downsample) ]
+        salmon_index            // channel: [mandatory] [ path(salmon_index) ]
+        ch_genome_fasta         // channel: [mandatory] [ val(meta), path(fasta) ]
```

Comment:
Nice additions ⭐ 
You should make a simple search and replace for `patrameter` -> `parameter` in this and other files.

Reply:
Good catch! It's so easy to have sp 😅 
I have applied Felix's suggestion and I have substituted all parameter (and also patrameter ) for string/boolean... on the next PR